codingtest_ajw/3일차/문자열-다루기-기본.js

```diff
@@ -0,0 +1,9 @@
+function solution(s) {
+    let answer = true;
+    if((s.length===4||s.length===6)&&isNaN(s)===false){
+        answer=true
+    }else{
+        answer=false
+    }
+    return answer;
```

Comment:
```suggestion
        return answer = true
    }
    return answer = false;
```
여기서 리턴을 바로 해주면 else가 필요 없어집니다. 당연히 아시겠지만 return이 실행되면 함수 실행이 끝납니다.